clang/test/Driver/sycl-device.cpp

```diff
@@ -22,3 +22,10 @@
 // RUN:   %clang -### -fsycl %s 2>&1 \
 // RUN:   | FileCheck -check-prefix=CHECK-SYCL-STD_VERSION %s
 // CHECK-SYCL-STD_VERSION: clang{{.*}} "-sycl-std=2017"
+
+/// Check that -aux-triple is set correctly
+// RUN:   %clang -### -fsycl -target aarch64-linux-gnu %s 2>&1 \
+// RUN:   | FileCheck -check-prefix=CHECK-SYCL-AUX-TRIPLE %s
+// CHECK-SYCL-AUX-TRIPLE-NOT: clang{{.*}} "x86_64-unknown-linux-gnu"
```

Comment:
What is the value of `CHECK-SYCL-AUX-TRIPLE-NOT` checks?

I think this test doesn't make sense on aarch64, right?

Reply:
yeah this test is trivially true on when running non x86 systems.
there doesn't seem to be a way to query the the triple of the device running the test.
after taking a look at what other cross-compilation test do they don't seem to have check like this so i removed the check.